docs/2.7.0/docs/daml-finance/tutorials/instrument-modeling/generic-extension.rst

```diff
@@ -128,22 +133,37 @@ It is now possible to create the *Election*:
   :start-after: -- CREATE_ELECTION_BEGIN
   :end-before: -- CREATE_ELECTION_END
 
+Note: these parts (election offer and election candidate) are not considered a core part of the Daml
+Finance library. There can be different processes to create the Election, so this is rather
+application specific. Still, in order to showcase one way how this could be done, this workflow is
+included here for convenience.
+
+The :ref:`Election <module-daml-finance-interface-lifecycle-election-24570>`
+has a flag *electorIsOwner*, which indicates whether the election is on behalf of the owner of the
+holding. This is typically the case for options, where the option holder has the right, but not the
+obligation, to exercise the option. On the other hand, for callable bonds it is not the holding
+owner (the bond holder) who gets to decide whether the bond is redeemed early. Instead, it is the
+counterparty. In this case, *electorIsOwner* would be false.
+
 A lifecycle rule is required to specify how to process the Election:
 
 .. literalinclude:: ../../src/test/daml/Daml/Finance/Instrument/Generic/Test/EuropeanOption.daml
   :language: daml
   :start-after: -- CREATE_LIFECYCLE_RULE_BEGIN
   :end-before: -- CREATE_LIFECYCLE_RULE_END
 
+This is similar to time-based lifecycling.
+
 Finally, it is possible to apply the Election according to the lifecycle rule provided:
 
 .. literalinclude:: ../../src/test/daml/Daml/Finance/Instrument/Generic/Test/EuropeanOption.daml
   :language: daml
   :start-after: -- APPLY_ELECTION_BEGIN
   :end-before: -- APPLY_ELECTION_END
 
-This create lifecycle effects, which can be claimed and settled in the usual way. This is described
-in :doc:`Getting Started: Lifecycling <../getting-started/lifecycling>`.
+This creates lifecycle effects, which can be claimed and settled in the usual way (as described in
+:doc:`Getting Started: Lifecycling <../getting-started/lifecycling>`). The value of *electorIsOwner*
```

Comment:
I don't think that's the case? 

The flag just controls which party makes the election, but the effects are generated through lifecycling (where the `electorIsOwner` flag does not play any role

Reply:
If it is misleading/incorrect, I could take it out.
Or you could propose an alternative formulation, as you prefer.

Reply:
I could suggest the following

This creates lifecycle effects, which can be claimed and settled in the usual way (as described in
:doc:`Getting Started: Lifecycling <../getting-started/lifecycling>`).  However,  the holding contract used to claim the effect must be compatible with the election that has been made: if `alice` made an election and `electorIsOwner = True`, then only a holding where `owner = alice` will be accepted.

If you think it makes the description more confusing, then I would take out the last sentence and stop at "can be claimed in the usual way"

Reply:
I think this makes sense. I have changed it accordingly.